test-utils/test-utils-persistence/src/main/java/uk/gov/justice/services/test/utils/persistence/DatabaseCleaner.java

```diff
@@ -46,6 +50,16 @@ public void cleanEventLogTable(final String contextName) {
         }
     }
 
+    public void cleanViewStoreTables(final String contextName, final String tableName, final String... tableNames) {
```

Comment:
Is there any reason why we still need the 2nd 'tableName' as a second argument?

It isn't very clear why it can't be rolled into the vararg